lib/screens/onboarding/show_onboarding_choice.dart

```diff
@@ -106,31 +106,42 @@ class ShowOnboardingChoice extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Column(
-      mainAxisAlignment: MainAxisAlignment.spaceAround,
-      children: <Widget>[
-        buildGroup(
-          'If you have an account\nclick here'.i18n,
-          'Import private key'.i18n,
-          onImport,
-        ),
-        FirebaseRemoteConfigService().featureFlagGuardiansEnabled
-            ? buildGroup(
-                'If you forget your private key\nclick here'.i18n,
-                'Recover account'.i18n,
-                onRecover,
-              )
-            : SizedBox.shrink(),
-        buildGroup(
-          'If you have an invite\nclick here'.i18n,
-          "Claim invite code".i18n,
-          onInvite,
-        ),
-        Container(
-          height: 10,
-        ),
-        // buildBottom()
-      ],
+    return SingleChildScrollView(
```

Comment:
This is fair enough as a scroll view - did you test this?

Reply:
Yes it was tested on small devices, Iphone 12,

before:
![image](https://user-images.githubusercontent.com/51983923/104625853-353eef00-5663-11eb-8dc5-7460fbad06ea.png)

after:
![image](https://user-images.githubusercontent.com/51983923/104625808-2a845a00-5663-11eb-9874-637cd04a4add.png)

after:
![image](https://user-images.githubusercontent.com/51983923/104625886-3d972a00-5663-11eb-9ceb-3fe757cedad7.png)

Reply:
Recover account is currently under a flag so it wont be displayed until key guardian feature is complete